client/app/bundles/HelloWorld/components/Filters.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default class Filters extends React.Component {
+    static propTypes = {
+        cuisines: PropTypes.array.isRequired,
+        updateCuisineFilter: PropTypes.func.isRequired,
+        updateRatingFilter: PropTypes.func.isRequired
+    };
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            cuisines: this.props.cuisines
+        };
+        this.state.cuisines.unshift('All');
+    }
+
+    handleCuisineFilterChange = (e) => {
+        this.props.updateCuisineFilter(e.target.value);
+    };
+
+    handleRatingFilterChange = (e) => {
+        this.props.updateRatingFilter(e.target.value);
+    };
+
+    render () {
+        let lines = this.state.cuisines.map((cuisine) => {
+            return <option key={cuisine} value={cuisine}>{cuisine}</option>
+        });
+        return (
+            <div className="filters">
```

Comment:
Why don't you use `sass-loader`: https://github.com/webpack-contrib/sass-loader ?